parcel/Example_NMoment_condensation.jl

```diff
@@ -0,0 +1,178 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import Cloudy as CL
+import Cloudy.ParticleDistributions as CPD
+import CloudMicrophysics.Parameters as CMP
+import CloudMicrophysics.MicrophysicsFlexible as CMF
+import ClimaParams as CP
+
+FT = Float64
+
+"""
+    ODE problem definitions
+"""
+function parcel_model_cloudy(dY, Y, p, t)
```

Comment:
I'll think about how to unify the OG parcel model with the Cloudy parcel model. But it's ok to have two separate ones for now. My only concern is that we won't see anywhere if this example stops working. So you could add the plot to the docs somewhere (then the docs will break if things break). Or move it to tests.